docusaurus/src/components/HeaderDecoration.jsx

```diff
@@ -329,6 +343,7 @@ export const HeaderDecoration = ({
   const isOss = boolStringToBool(isOssString);
   const isCloud = boolStringToBool(isCloudString);
   const isPypiPublished = boolStringToBool(isPypiPublishedString);
+  const isEnterprise = boolStringToBool(isEnterpriseString);
```

Comment:
I don't see `isEnterpriseString` defined anywhere

Reply:
FYI enterprise-ness could be inferred from the connector's license type, if you have it available

Reply:
> I don't see isEnterpriseString defined anywhere

This is defining the react props / inputs to this method.  It's passed in via the remark plugins.

> FYI enterprise-ness could be inferred from the connector's license type, if you have it available

Yeah! Once we get the connector really in the metadata service (e.g. not a custom connector), all of that information will be available automatically... but today I just hard-code it to true (in the remark plugin, based on the markdown file path with the `/enterprise-connectors/`)